iidm/iidm-api/src/main/java/com/powsybl/iidm/network/ThreeSides.java

```diff
@@ -0,0 +1,7 @@
+package com.powsybl.iidm.network;
+
+public enum ThreeSides {
```

Comment:
I am not sure it is the good PR to introduce that. @geofjamg what do you think ? I was thinking about a dedicated PR to rework on sides in general: HVDC line, measurements, terminal ref, transfromers. And if we do, I prefer `GenericSide`.

Reply:
We discussed this. ThreeSides in the iidm API is needed to avoid strange dependencies to a Side defined in TerminalRefXml everywhere because we remove TerminalRef from iidm API where it was declared before.
The full rework for the SIDE enum will take place in another PR indeed.
As for the name keep in mind we will probably need two generic enums "TwoSides" and "ThreeSides" used everywhere and chosen with the equipement type. Generic conversion function between the two enums will also be added.

Reply:
We will continue this Side enum work in a dedicated PR. Here we have just the minimal addition for this work.